### PR TITLE
updated 'raise and issue' link in Readme 

### PR DESCRIPTION
**Pull request by heidinett**
updated 'raise and issue' link in Readme to point to new location of repo.
Fixes issue #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
  <img src="docs/assets/ACCESS_ATMOSPHERE_icon.png" height="300">   
 </div>
 
-<a href="https://atmospheric-cookbook.readthedocs.io/en/latest/">
+<a href="https://atmosphere-cookbook.readthedocs.io/en/latest/">
```
Comment (navidcy) [inline README.md:5]: I don't think this link is working as corrected here

Comment (heidinett) [reply README.md:5]: No, it is not. Is it possible to change the URL of the readthedocs from `https://atmospheric-cookbook.readthedocs.io/en/latest/` to `https://atmosphere-cookbook.readthedocs.io/en/latest/` without recreating a new one?

Comment (navidcy) [reply README.md:5]: Possibly. Do you have a readthedocs account? I can add you as a maintainer.

Comment (heidinett) [reply README.md:5]: No I don't have a readthedocs account.

Comment (heidinett) [reply README.md:5]: Just signed up for an account. Happy to be added as a maintainer.

Comment (navidcy) [reply README.md:5]: added you

Comment (heidinett) [reply README.md:5]: ok, thanks. Is it possible to change the project slug or do we need to make a new project with the new name?

Comment (navidcy) [reply README.md:5]: I tried to change it and didn't find how... but that doesn't mean it's not possible. There is a bit chance I just didn't figure it out.

Comment (navidcy) [reply README.md:5]: chatGPT claims that we can -- however I couldn't see any "Advanced Settings" option...

To change the project slug for a Read the Docs project, you'll need to go through a few steps on the Read the Docs website:

### Steps to Change Read the Docs Project Slug

1. **Log in to Read the Docs**: Go to [Read the Docs](https://readthedocs.org/) and log in with your account.

2. **Access Your Project**: From the dashboard, select the project you want to modify.

3. **Navigate to Project Admin**:
   - Click on **"Admin"** in the project's menu to go to the project's admin settings.

4. **Change the Project Slug**:
   - Under the **"Advanced Settings"** section, look for the field labeled **"Project Slug."**
   - Update the slug to the desired value. 
   - Note that the slug must be unique and can only contain letters, numbers, hyphens (`-`), and underscores (`_`).

5. **Save the Changes**: Click the **"Save"** button at the bottom of the page.

### Important Considerations

- **URL Changes**: Changing the project slug will change the URL of your documentation, so make sure to update any links pointing to the old URL.
- **Builds and Webhooks**: If you have external integrations, webhooks, or automation scripts, check to ensure they still work correctly after the slug change.

Would you like more details on any specific step?

Comment (heidinett) [reply README.md:5]: Got to Step 4. - I cannot see any "advanced Settings" under "Admin" in the project's menu.

Comment (navidcy) [reply README.md:5]: Same

Comment (navidcy) [reply README.md:5]: We can leave it as is for now?

Comment (heidinett) [reply README.md:5]: ok, sure - we can leave it for now. 
FYI - I sent an email to support@readthedocs.org asking whether it is possible to change the URL of the project slug.

**Comment by heidinett** [comment]
Reverted readthedocs link back to "atmospheric-cookbook" instead of "atmosphere-cookbook" until we can change the URL of project slug.

**Review by navidcy — APPROVED** [review]

**Pull request merged**